lib/Dialect/SPIRV/SPIRVOps.cpp

```diff
@@ -199,6 +199,162 @@ static void printNoIOOp(Operation *op, OpAsmPrinter *printer) {
   printer->printOptionalAttrDict(op->getAttrs());
 }
 
+//===----------------------------------------------------------------------===//
+// spv.AccessChainOp
+//===----------------------------------------------------------------------===//
+
+// Function to extract an actual value from ConstantOp.
+static std::pair<StringRef, bool> extractValueFromConstOp(Operation *op,
+                                                          int32_t &indexValue) {
+  // spv.constant operation contains kValueAttrName.
+  auto valueAttr = op->getAttr(kValueAttrName);
```

Comment:
We should actually `dyn_cast<spirv::ConstantOp>()` here to make sure this is actually from a constant op, not some other op with `value` attribute. And then we can use the `op.value()` accessor.

Reply:
Oh, I see, thanks. Fixed.